code/src/components/TaskList.js

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import moment from 'moment';
+import tasks from 'reducers/tasks';
+import styled from 'styled-components';
+import { FcEmptyTrash } from 'react-icons/fc';
+
+const StyledTasklist = styled.div`
+display: flex; 
+word-wrap: break-word;
+flex-direction:column;
+background:#a8e6cf;
+border-radius:  10%;
+box-shadow: rgba(0, 0, 0, 0.15) 0px 15px 25px, rgba(0, 0, 0, 0.05) 0px 5px 10px;
+margin-bottom: 10%; 
+input[type='checkbox'].checkbox {
+    --size: 20px;
+    --radius: calc(var(--size) / 10);
+    --bg: #ff8b94;
+    --border-radius:5px;
+    --color: white;
+    --time: 0.5s;
+}
+
+@media (min-width:1024px) {
+  min-height: 80vh;
+}
+
+@media (max-width:1024px) {
+  min-height: 60vh;
+}
+`
+
+const StyledList = styled.ul`
+display: flex;
+flex-wrap:wrap;
+justify-content: center;
+list-style-type: none;
+margin: 0;
+padding: 0;
+
+`
+
+const StyledListItem = styled.li`
+font-family: 'Poppins', sans-serif;
+font-size: 20px;
+background:#63ace5;
+color:#FDEDDE;
+padding: 20px;
+border-radius: 15px;
+margin: 20px 30px; 
+width: 98%;
+height: auto;
+box-shadow: rgba(0, 0, 0, 0.15) 0px 15px 25px, rgba(0, 0, 0, 0.05) 0px 5px 10px;
+
+@media (max-width:350px){
+  font-size: 14px;
+  width:60%;
+}
+
+@media (max-width:667px){
```

Comment:
I would consider making some minor changes to the layout on mobile view, so that you can fit a bit more on the screen. Perhaps the title section can be a bit smaller? That way there would be a little less scrolling if you (like me) have a lot of todo's in a day. That's my personal preference, your app still works excellent on mobile just as is too.